### DISCUSSION TITLE
Connector defaults

### DISCUSSION THREAD

**Discussion by JonPolish** [freeplane/freeplane#973] [Help] [answered]
Can the default connector style be changed? I would like to use straight lines with arrows. 

Thank you.

Jon

**Comment by abc16361** [comment, marked as answer]
You can set up styles on the style manager (Format->Manage Styles -> Edit styles). Open it, right-click on Default or user defined styles, choose  'Add connector style' on context menu. When the connector comes out, right click, then you can set it. It's the way I do.
![connectorStyle](https://user-images.githubusercontent.com/101362122/215686647-20e48336-fc4a-4fe1-9854-1791e0baa31b.png)
![connectorStyle1](https://user-images.githubusercontent.com/101362122/215686650-f200ed46-dc19-45ea-9145-34f9c5b69eb0.png)

**Reply by JonPolish** [reply]
Thank you so much.

Jon
